src/components/CustomChannelComponent.tsx

```diff
@@ -134,10 +134,10 @@ export function CustomChannelComponent(props: CustomChannelComponentProps) {
   }, [lastMessage?.data]);
 
   const dynamicReplyOptions =
-    lastMessage?.extendedMessage != null &&
-    'suggested_replies' in lastMessage.extendedMessage &&
-    lastMessage.extendedMessage.suggested_replies != null
-      ? JSON.parse(lastMessage.extendedMessage.suggested_replies)
+    lastMessage?.extendedMessagePayload != null &&
+    'suggested_replies' in lastMessage.extendedMessagePayload &&
+    lastMessage.extendedMessagePayload.suggested_replies != null
+      ? lastMessage.extendedMessagePayload.suggested_replies
```

Comment:
Should we update the Chat SDK version?

Reply:
How about creating a utility function for getting suggested_replies?

```ts
  const dynamicReplyOptions = getSuggestedReplies(lastMessage);

  // src/utils/messages.ts
  function getSuggestedReplies(message?: ClientUserMessage): string[] {
    if (Array.isArray(message?.extendedMessagePayload?.suggested_replies)) {
      return message.extendedMessagePayload.suggested_replies;
    } else {
      return [];
    }
  }
```

Reply:
Yeah right I was waiting to be released. Just updated the versions! https://github.com/sendbird/chat-ai-widget/pull/65/commits/447ff61d5e438e7bd9770de9ad9d81cd9e8a7bb8

Reply:
Oh I don't know why but I missed your 2nd comment. I wasn't there when I saw the 1st one 😅
Let me address it in another PR.

Reply:
We have a chance here https://github.com/sendbird/sendbird-uikit-react/pull/828 lol

Reply:
Yeah let me push a commit there. Thanks!